test/files/97-requests-cut.t

```diff
@@ -0,0 +1,9 @@
+>>>
+GET /?p=%20 HTTP/1.0
+User-Agent: Mozilla
+
+G
+>>>
+ET /?p=%21 HTTP/1.0

```

Comment:
If I remember correctly request pipelining is only supported by HTTP/1.1.

Reply:
You are right
Do you want me to update the test case ?

Reply:
Well I guess I'm more surprised about this actually working. Are we simply supporting pipelining for 1.0 as well? This is not RFC compliant as far as I know, but maybe it is seen in the wild?

Reply:
Yes simply supporting pipelining no matter the protocol version